week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getWordLength(word) {
-  return "word".length()
+  return word.length;
 }
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+ 
+  return(a*b*c);
```

Comment:
On this line you've got 
```suggestion
  return a * b * c;
```
Try to space out the variables and symbols on this line as above.